src/dojos/cd-dojo-details.vue

```diff
@@ -305,8 +332,45 @@
       border-bottom: 1px solid #bebebe;
       padding-bottom: 8px;
 
-      ~ .cd-dojo-details__heading {
-        margin-top: 64px;
+      &-label {
+        float: right;
+        font-size: 12px;
```

Comment:
use variables : 
https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L52

Reply:
should we change this everywhere?

Reply:
if @DanielBrierton is OK with what I'm saying :)